foundry/external_data_architectures.py

```diff
@@ -25,3 +26,23 @@ def __getitem__(self, idx):
         
         return item
     
+class FoundryDataset_TF(Sequence):
+    def __init__(self, inputs, targets):
+        self.inputs=inputs
+        self.targets=targets
+
+    def __len__(self):
+        return len(self.inputs[0])
+
+    def __getitem__(self, idx):
+        item = {"input": [], "target": []}
+        
+        for input in self.inputs:
+            item["input"].append(np.array(input[idx]))
+        item["input"] = np.array(item["input"])
+        
+        for target in self.targets:
+            item["target"].append(np.array(target[idx]))
+        item["target"] = np.array(item["target"])
+        
+        return item
```

Comment:
Do we have Flake8 installed for the CI? If so, it should point out style problems like the missing line on some of these files.